src/vs/workbench/contrib/terminal/browser/widgets/terminalQuickFixMenuItems.ts

```diff
@@ -62,8 +63,10 @@ export function toMenuItems(inputQuickFixes: readonly TerminalQuickFix[], showHe
 function getQuickFixIcon(quickFix: TerminalQuickFix): { codicon: Codicon } {
 	switch (quickFix.type) {
 		case TerminalQuickFixType.Opener:
-			// TODO: if it's a file link, use the open file icon
-			return { codicon: Codicon.link };
+			if ('uri' in quickFix.action && quickFix.action.uri?.scheme) {
```

Comment:
I think scheme is required?

https://github.com/microsoft/vscode/blob/3b4e310d429412f061325912c8673bbe186739a2/src/vs/base/common/uri.ts#L121